packages/gatsby-source-shopify/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @gatsbyjs/integrations-collaboration
```

Comment:
If we were to add a `CODEOWNERS` file again, we'd add it in the root of the repo. But this needs to be discussed as we removed it some time ago on purpose

Reply:
Gotcha, I will remove this file!